tiled/storage.py

```diff
@@ -92,6 +109,8 @@ def parse_storage(item: Union[Path, str]) -> Storage:
         result = FileStorage(item)
     elif scheme == "postgresql":
         result = SQLStorage(item)
+    elif scheme == "bucket":
```

Comment:
There won't be a bucket scheme. Instead, this function must be extended to accept `dict` (a YAML object from config). While we're here, we might accept a dict for  PostgreSQL as well as for Blob/Bucket.

We currently accept SQL creds like this only:
```yaml
- postgresql://username:password@host:port/database
```

but we could additionally accept a more structured input:

```yaml
- uri: postgresql://host:port/database
  username: username
  password: password
```

And buckets would of course be similar.